src/gui/Gui.java

```diff
@@ -142,12 +147,13 @@ public synchronized void run() {
         gf.setDoubleBuffered(true);
 
         jf.setResizable(false);
-        jf.setSize(scale*w.getXyWorld()[0].length, scale*w.getXyWorld().length + frameSize);
+        jf.setSize(scale*w.getXyWorld()[0].length, scale*w.getXyWorld().length + frameSize + 50);
         jf.setLocationRelativeTo(null);
         jf.addKeyListener(new KeyControl(p));
         jf.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
         jf.setIconImage(new ImageIcon(iconPath.toString()).getImage()); //img als icon
-
+        jL.setText(""+score.getScore());
```

Comment:
_Exception in thread "Thread-1" java.lang.NullPointerException
	at src.gui.Gui.run(Gui.java:155)_

Pacman.score is not initialized before being called in Gui